### ISSUE TITLE
heroku kafka addon

### ISSUE THREAD

**Issue by minervax** [jamesward/hello-kafka-salesforce#1]
Would be nice to note that adding kafka to hero costs 1500$/month

**Comment by jamesward** [comment]
Yeah, good point.  There was a limited beta that was free but I think now that it is GA the base plans are pretty costly.  Hopefully there will be free developer plans at some point.
